.github/workflows/cypress-integration-tests.yml

```diff
@@ -18,11 +18,11 @@ on:
   push:
     branches: [main]
     paths-ignore:
-      - 'openmetadata-docs/**'
+      - "openmetadata-docs/**"
```

Comment:
Please revert the quotes here!

Reply:
Let's only add new env Secrets and not update anything in this yaml file!

Reply:
seems like an auto-formatting did its job 😅, ill update